src/scd.cpp

```diff
@@ -246,6 +247,25 @@ namespace openre::scd
         return SCD_RESULT_NEXT_TICK;
     }
 
+    // 0x004E43D0
+    static int scd_evt_end(SCE_TASK* sce)
+    {
+        auto eax = sce->Sub_ctr;
+        // loc_4E4407
+        if (eax == 0)
+        {
+            sce->Status = 0;
+            return SCD_RESULT_NEXT_TICK;
+        }
+
+        auto edx = *(&sce->Task_level + eax);
+        auto ecx = eax - 1;
```

Comment:
Again try to name these. `ecx` could be some last, `callerIndex`. Not sure exactly what edx is here, some kind of offset into the SCD stack.